usaspending_api/etl/executive_compensation_etl.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from usaspending_api.references.models import LegalEntity, LegalEntityOfficers
+from usaspending_api.etl.broker_etl_helpers import dictfetchall, PhonyCursor
+
+logger = logging.getLogger("console")
+
+# Broker SQL query
+FILE_E_QUERY = """
+SELECT *
+FROM executive_compensation e
+INNER JOIN (
```

Comment:
I believe the group by in the join performs a similar function as a DISTINCT ON but I can switch it anyway to be more explicit.